Sources/MySQL/Connection/PacketParser.swift

```diff
@@ -129,9 +129,9 @@ internal struct MySQLPacketParser: ByteParser {
     fileprivate func parseHeader(from buffer: ByteBuffer) -> Int? {
         guard buffer.count >= 3 else { return nil }
         
-        let byte0: UInt32 = numericCast(buffer[0])
-        let byte1: UInt32 = numericCast(buffer[1])
-        let byte2: UInt32 = numericCast(buffer[2])
+        let byte0: UInt32 = (numericCast(buffer[0]) as UInt32).littleEndian
```

Comment:
Since you're immediately casting to a specific numeric type, it would be more concise here to write `let byte0 = UInt32(buffer[0])`. Additionally, the `.littleEndian` call is a no-op on all little-endian platforms (and thus all platforms supported by Swift).